src/gui/settingsdialog.cpp

```diff
@@ -84,13 +84,11 @@ QString shortDisplayNameForSettings(OCC::Account *account, int width)
 namespace OCC {
 
 SettingsDialog::SettingsDialog(ownCloudGui *gui, QWidget *parent)
-    : QDialog(parent)
+    : QDialog(parent, Qt::WindowFlags() & ~Qt::WindowContextHelpButtonHint)
     , _ui(new Ui::SettingsDialog)
```

Comment:
Should we also setAttribute(Qt::WA_DeleteOnClose, true); here?

Reply:
The widget is only initialised once so we don't want it to be deleted.

Reply:
But don't we want to delete it when we exit?